<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoneyManagement</class>
 <widget class="QMainWindow" name="MoneyManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1205</width>
    <height>759</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="exitMain">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>60</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>退出程序</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minimize_tuopan">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>130</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>最小化到托盘</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minimize_renwulan">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>200</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>最小化到任务栏</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1205</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>理财管理软件</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
